snapcraft/__init__.py

```diff
@@ -115,7 +115,11 @@
 import os
 import shutil
 
+from snapcraft._help import topic_help  # noqa
+from snapcraft._store import login, logout, upload  # noqa
```

Comment:
With tupling, but it would also be an indication that we are doing something wrong IMHO